conf/ocsci/ocp-4.3-ga-upgrade.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Use this config for upgrading of OCP 4.3 GA cluster
+UPGRADE:
+  ocp_upgrade_version: "4.3.3"
```

Comment:
this value should not be there as we want to just take this value dynamically as I pointed out.
You can maybe just do as commented value as example that ppl can change for some specific version here.